### ISSUE TITLE
Add --config -c option to arguments

### ISSUE THREAD

**Issue by k2662** [FelixKratz/JankyBorders#64]
we could just use the yabai implementation of it to simplify things.

**Comment by FelixKratz** [comment]
I dont think this is needed. You can pass all options to the primary process on startup.
